code/controllers/Processes/modifier_process.dm

```diff
@@ -0,0 +1,28 @@
+#define MODIFIER_PROCESS_INTERVAL	10
```

Comment:
Unless you need this elsewhere, just don't use a `#define` for it. No other processes have a define directive for their interval, they just use number literals.